README.md

```diff
@@ -1,73 +1,47 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Leaderboard App
 
+This project send data to the leaderboad api and get it from the same api to display the score when the person adds it
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://fibini.github.io/Leaderboard-App/dist/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+Github profile and code editor
 
 ### Setup
+copy the repository from the drop down code menu
 
 ### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+clone the repository to your local machine through your code editor using git clone
+git clone git@github.com:fibini/Leaderboard-App.git
+> cd Leaderboard
+> npm install
+> code
+> npm start
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.